package.json

```diff
@@ -4,19 +4,25 @@
   "type": "module",
   "license": "MIT",
   "scripts": {
-    "dev": "npx prisma db push && nodemon",
-    "build": "npx prisma generate && npx swc ./src -d dist",
-    "start": "npx prisma migrate deploy && node dist/index.js",
+    "dev": "prisma migrate dev && tsx watch src/index.ts",
```

Comment:
Why change this? Please keep `nodemon` here.

Reply:
I think nodemon is an extra dependency we don't need. We can use `--watch` flag from node v18+ and is more efficient from nomemon. This is my opinion of cource and I want to talk about it.